### PR TITLE
Added the rest of 1.16.1 data

### PR DESCRIPTION
**Pull request by AppDevMichael**

### CODE REVIEW

File: data/pc/1.16.1/blockCollisionShapes.json [hunk 1]
```diff
@@ -0,0 +1,1938 @@
...
+        "honey_block": 64,
+        "honeycomb_block": 1,
+        "nether_gold_ore": 1,
+        "soul_fire": 0,
+        "soul_soil": 1,
+        "basalt": 1,
+        "polished_basalt": 1,
+        "soul_torch": 0,
+        "soul_wall_torch": 0,
+        "chain": 270,
```
Comment (Karang) [inline data/pc/1.16.1/blockCollisionShapes.json:690]: The shapes array stop at 269. Could you add an entry for the chain block ?

Comment (AppDevMichael) [reply data/pc/1.16.1/blockCollisionShapes.json:690]: I did at the bottom didn't I?

Comment (AppDevMichael) [reply data/pc/1.16.1/blockCollisionShapes.json:690]: Bruh it disappeared

Comment (Karang) [reply data/pc/1.16.1/blockCollisionShapes.json:690]: thanks

File: data/dataPaths.json [hunk 2]
```diff
@@ -555,7 +555,8 @@
       "recipes": "pc/1.16.1",
       "instruments": "pc/1.16.1",
       "entities": "pc/1.16.1",
-      "windows": "pc/1.16.1"
+      "windows": "pc/1.16.1",
+      "blockCollisionShapes": "pc/1.16.1"
```
Comment (rom1504) [inline data/dataPaths.json:559]: Can you use materials and language too there ?

**Pull request merged**